data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/Event.java

```diff
@@ -85,4 +86,10 @@
      * @since 1.2
      */
     boolean isValueAList(String key);
+
+    /**
+     * @return a Map representation of the Event
+     * @since 1.3
+     */
+    Map<Object, Object> getAsMap();
```

Comment:
I'd avoid using a `get` method here since it sounds like a getter. You could rename this to either `toMap()` or `asMap()`. I'd prefer `toMap()` because it is consistent with other Java APIs.